libs/orchestrator-ui-components/src/lib/components/Table/columns.ts

```diff
@@ -1,20 +1,25 @@
 import { EuiBasicTableColumn } from '@elastic/eui';
 
+// Todo need to Pick a few props from EuiBasicTableColumn to prevent none-functioning props (truncateText)
```

Comment:
Is there a ticket for this? How do we keep track of TODO's?

Reply:
In general, you will probably not leave TODO's in the code, only NOTE's.

Reply:
My take is: If the TODO is not fixed in the same PR, make an issue and put the issue number in the TODO.

Reply:
Some todo's are not in scope of the story, for now I create issues and link to them

Reply:
Added a new issue to backlog 
https://github.com/workfloworchestrator/orchestrator-ui/issues/130